stubs/pydu/request.pyi

```diff
@@ -12,3 +12,4 @@ class FileName(object):
 def download(url: str, dst: str=None) -> str: ...
 def check_connect(ip: str, port: int, retry: int=1, timout: float=0.5) -> Optional[str]: ...
 def update_query_params(url: str, params: dict) -> str: ...
+def cookies_string_to_dict(cookies_str:str)->dict: ...
```

Comment:
Please be careful about formatting. You was missing blank between `cookies_str:` and `str`